yaml/wp_energy_cop.yaml

```diff
@@ -0,0 +1,32 @@
+## WHAT IS COP ################################################################### 
+# The COP value, or “Coefficient of Performance,” indicates the ratio of         #
+# the heat generated by the heat pump to the driving energy (electricity)        #
+# required for it. Simply put, it measures how many units of heat a heat pump    #
+# can produce from one unit of electricity and deliver to the heating circuit.   #
+# This is very interesting for owners because a heat pump does not require       #
+# fossil fuels. Their heating costs are solely determined by                     #
+# electricity consumption.                                                       #
+#                                                                                #
+# In general, the higher the COP, the more efficient the heat pump is.           #
+# COP = Heat delivered to the heating circuit​ / Energy used (electricity)        #
+##################################################################################
+
+defaults:
+  interval: $interval_slow
+
+sensor:
+  - platform: template
+    name: ${property}
+    id: ${property}
+    icon: "mdi:home-analytics"
+    state_class: "measurement"
+    accuracy_decimals: 2
+    update_interval: ${interval}
+    filters:
```

Comment:
I think the throttling is not needed, since the update_interval dictates how often the lambda is called